File: RxTools-library/src/main/java/com/vondear/rxtools/model/Gps.java
package com.vondear.rxtools.model;

/**
 * @author Vondear
 * @date 2017/6/19
 */

public class Gps {

    private double mLatitude;
    private double mLongitude;

    public Gps() {
    }

    public Gps(double longitude, double mLatitude) {
        setLatitude(mLatitude);
        setLongitude(longitude);
    }

    public double getLatitude() {
        return mLatitude;
    }

    public void setLatitude(double latitude) {
        this.mLatitude = latitude;
    }

    public double getLongitude() {
        return mLongitude;
    }

    public void setLongitude(double longitude) {
        this.mLongitude = longitude;
    }

    @Override
    public String toString() {
        return mLongitude + "," + mLatitude;
    }
}

File: RxKit/src/main/java/com/tamsiree/rxkit/model/Gps.kt
package com.tamsiree.rxkit.model

/**
 * @author Tamsiree
 * @date 2017/6/19
 */
class Gps {
    var latitude = 0.0
    var longitude = 0.0

    constructor()
    constructor(mLongitude: Double, mLatitude: Double) {
        latitude = mLatitude
        longitude = mLongitude
    }

    override fun toString(): String {
        return "$longitude,$latitude"
    }
}